.circleci/config.yml

```
@@ -4,7 +4,7 @@ version: 2.1
 setup: true
 
 orbs:
-    gravitee: gravitee-io/gravitee@2.0
+    gravitee: gravitee-io/gravitee@2.1
```

you could even use the version 2.2
```suggestion
    gravitee: gravitee-io/gravitee@2.2
```